src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -977,6 +977,30 @@ public async Task SendUnverifiedOrganizationDomainEmailAsync(IEnumerable<string>
         await _mailDeliveryService.SendEmailAsync(message);
     }
 
+    public async Task SendInitiateDeleteOrganzationEmailAsync(string email, Organization organization, string token)
+    {
+        var message = CreateDefaultMessage("Delete Your Organization", email);
```

Comment:
Feedback from product:
```suggestion
        var message = CreateDefaultMessage("Request to Delete Your Organization", email);
```